docs/src/props-table.js

```diff
@@ -27,15 +33,28 @@ function PropsTable({children}) {
   )
 }
 
-function Row({name, type, defaultValue, description, required}) {
+function Row({name, type, defaultValue, description, required, deprecated}) {
```

Comment:
Doesn't look like this PR adds usage of `deprecated` — that's fine by me; I'm just calling it out in case that wasn't intended.

Reply:
Yup. I don't think we have many deprecated props. But it felt like something the `Row` component should support